### PR TITLE
Card should accept borderWidth={0}

### PR DESCRIPTION
**Pull request by sdalonzo**
https://github.com/pricelinelabs/design-system/issues/383

### CODE REVIEW

File: src/__tests__/Card.js [hunk 1]
```diff
@@ -90,4 +90,10 @@ describe('Card', () => {
       '0 0 2px 0 rgba(0,0,0,.08),0 2px 8px 0 rgba(0,0,0,.16)'
     )
   })
+
+  test('renders border 0 without warning', () => {
+    const json = renderer.create(<Card borderWidth={0} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('border', '0')
```
Comment (jxnblk) [inline src/__tests__/Card.js]: This isn't actually testing whether the console warning is being fired or not. I think we have some other places where we use `jest.fn()` to test prop types

File: src/Card.js [hunk 2]
```diff
@@ -36,7 +37,7 @@ Card.propTypes = {
   boxShadowSize: PropTypes.oneOf(['sm', 'md', 'lg', 'xl']),
   borderColor: PropTypes.string,
   ...propTypes.borderRadius,
-  borderWidth: PropTypes.oneOf([1, 2])
+  borderWidth: PropTypes.oneOf([0, 1, 2])
```
Comment (BeniCheni) [inline src/Card.js]: Is there actually a use case where a `Card` would have "_clear/transparent_" border?

(Confirming if the original code actually intended to force all `Card`s to have at least a 1px border.)

File: src/__tests__/Card.js (hunk 1 above)
Comment (jxnblk) [reply src/__tests__/Card.js]: Are you planning to update the test here like you did in #416?

Comment (sdalonzo) [reply src/__tests__/Card.js]: yes, sorry, just been busy

Comment (sdalonzo) [reply src/__tests__/Card.js]: pushed a similar use of console.error.mock.calls

**Comment by sdalonzo** [comment]
@jxnblk made that update to check the calls of the mocked console.error. I think this should be good to go now

**Comment by sdalonzo** [comment]
Will consider merge at our meeting tomorrow

**Pull request merged**